src/transactions/feed/TransactionFeed.tsx

```diff
@@ -63,8 +64,10 @@ function TransactionFeed() {
     return groupFeedItemsInSections(pendingTransactions, confirmedFeedTransactions)
   }, [pendingTransactions, confirmedFeedTransactions])
 
+  // TODO(ACT-1006): Replace with real feature gate
+  const dummyGetStartedFlag = false
```

Comment:
if you instead imported a function returning the feature flag value (can still be hardcoded to false), you could more easily mock the feature flag in a test and verify that the correct variation of the UI is shown.

Reply:
I'll keep this in mind for the future, but since it's tested in the follow-up PR #4755 it seem worth adding to this PR.